modules/day-time.js

```diff
@@ -0,0 +1,10 @@
+import { DateTime } from './luxon.js';
+
+export default function getTime() {
```

Comment:
- [ ] it would be better if you use the es6 `arrow function` instead of the `normal function` in order to practice to work with both